src/containers/Home.jsx

```diff
@@ -24,8 +24,8 @@ const styles = StyleSheet.create({
     marginBottom: 15,
     color: theme.colors.white
   },
-  link: {
-    ...theme.text.link
+  link_dark_bg: {
```

Comment:
ok - i'll see what @lady3bean thinks. this link isn't necessarily more important, but will defer to them for naming convention.